doc/dbus_api.md

```diff
@@ -288,3 +288,37 @@ when the question is answered.
 - Attempt -> unsigned 32-bit integer (r)
   Current attempt to decrypt the device. This value is useful for clients to know if the very same
   question is asked again (i.e., when the provided password did not work).
+
+
+## Status
+
+Each service exports an *status* object, for example `/org/opensuse/DInstaller/Software/Status1`.
+
+### org.opensuse.DInstaller.Status1
+
+#### Properties
+
+- Id -> unsigned 32-bit integer (r)
+  Id of the status (0 error, 1 probing, etc).
```

Comment:
Just here I expect extensive list of enums. It is API documentation, so all values should be specified here.